pkg/metrics/metrics.go

```diff
@@ -14,11 +15,25 @@ const (
 	handlerNameLabel    = "handler_name"
 	hasErrorLabel       = "has_error"
 
+	contextLabel = "ctx"
 	groupLabel   = "group"
 	versionLabel = "version"
 	kindLabel    = "kind"
+
+	contextIDKey = "cache_context_id"
```

Comment:
While it's very unlikely to cause a clash, the best practices  seem to have formed around non-string keys for context keys?